src/ucp/dt/dt.h

```diff
@@ -41,4 +42,26 @@ typedef struct ucp_dt_state {
 size_t ucp_dt_pack(ucp_datatype_t datatype, void *dest, const void *src,
                    ucp_dt_state_t *state, size_t length);
 
-#endif
+static UCS_F_ALWAYS_INLINE void
+ucp_dt_clear_rndv_lanes(ucp_dt_state_t *state)
```

Comment:
maybe it is worth not to mention RNDV, as the code is rather generic and is in dt.h

Reply:
renamed